tensorflow/lite/micro/tools/ci_build/test_bazel.sh

```diff
@@ -17,24 +17,5 @@
 set -e
 set -x
 
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-ROOT_DIR=${SCRIPT_DIR}/../../../../..
-cd "${ROOT_DIR}"
-
-source tensorflow/lite/micro/tools/ci_build/helper_functions.sh
-
-# We are using a bazel build followed by bazel test to make sure that the CI
-# covers non-test binary targets as well. These were previousbly covered by
-# having build_test but that was removed with #194.
-
-CC=clang readable_run bazel build ... \
-  --config=ci \
-  --build_tag_filters=-no_oss
-CC=clang readable_run bazel test ... \
-  --config=ci \
-  --test_tag_filters=-no_oss --build_tag_filters=-no_oss \
-  --test_output=errors
-
-# TODO(b/178621680): enable ubsan once bazel + clang + ubsan errors are fixed.
-#CC=clang readable_run bazel test tensorflow/lite/micro/... --config=ubsan --test_tag_filters=-no_oss,-noubsan --build_tag_filters=-no_oss,-noubsan
-
+bazel test ... \
```

Comment:
`...` will only do from the current directory and below right? I think we should still include `cd root_dir`?

Reply:
Yes, `...` is from pwd down; however, CI jobs already run from the root_dir.

Reply:
I would lean toward just leaving the previous code for cd root_dir, its the most clear for a reader, rather than needing to check where CI runs from.